packages/react-core/src/components/Compass/CompassPanel.tsx

```diff
@@ -0,0 +1,55 @@
+import styles from '@patternfly/react-styles/css/components/Compass/compass';
+import { css } from '@patternfly/react-styles';
+
+interface CompassPanelProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content of the panel. */
+  children: React.ReactNode;
+  /** Additional classes added to the panel. */
+  className?: string;
+  /** Indicates the panel should have a pill border radius */
+  isPill?: boolean;
+  /** Indicates the panel should expand to fill the available height */
+  isFullHeight?: boolean;
+  /** Indicates the panel should scroll its overflow */
+  isScrollable?: boolean;
+  /** Indicates the panel should have no border */
+  hasNoBorder?: boolean;
+  /** Indicates the panel should have no padding */
+  hasNoPadding?: boolean;
+  /** Indicates the panel should have a gradient border */
+  hasGradientBorder?: boolean;
```

Comment:
This prop's styles conflict with the default styles of the panel. Since I'm not aware of a use case for this style on a panel, I'd just remove it? Also FWIW since gradient border isn't an arbitrary style, I'd name this type of prop based on the role, which is "ai indicator" from design (what the class in core is named). Clicking on something with this class should trigger some sort of AI interaction.